### DISCUSSION TITLE
Why was a Python requirement introduced in version 1.9.5?

### DISCUSSION THREAD

**Discussion by FSys77** [fluent/fluent-bit#6219] [General]
I was building my dockerized my FluentBit container since a couple of years now. No problems so was.

Base is a Ubuntu 20.04 LTS image (`FROM ubuntu:focal`).

The last working Fluent Bit version was 1.9.4 for me.

From version 1.9.5 onward it started complaining in the `cmake` phase:

```
/usr/bin/env: 'python3': No such file or directory
-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY OPENSSL_INCLUDE_DIR) 
```

Questions:
(1) Why is Python now required?
(2) What was changed regarding OpenSSL?

**Comment by patrick-stephens** [comment]
I think this is a duplicate of https://github.com/fluent/fluent-bit/issues/6217#issue-1411272126 so let's try to keep in one place.
